packages/web-app-files/src/components/FilesList/KeyboardActions.vue

```diff
@@ -88,42 +77,28 @@ export default defineComponent({
       addFileSelection: 'ADD_FILE_SELECTION'
     }),
 
-    handleSelectionShortcuts(event) {
+    handleShortcuts(event) {
       const key = event.keyCode || event.which
-      const ctrl = window.navigator.platform.match('Mac') ? event.metaKey : event.ctrlKey
       const shift = event.shiftKey
-
-      this.handleFileSelectionShortcuts(key, shift, ctrl, event)
-    },
-
-    handleClipboardShortcuts(event) {
-      const key = event.keyCode || event.which
       const ctrl = window.navigator.platform.match('Mac') ? event.metaKey : event.ctrlKey
-      const isCopyAction = key === 67
-      const isPasteAction = key === 86
-      const isCutAction = key === 88
       const isTextSelected = window.getSelection().type === 'Range'
-
+      const customKeyBindings = (window.getSelection().focusNode as HTMLElement)?.closest(
```

Comment:
Make sure to have an empty folder and then use the keyboard: throws a lot of errors (see browser console) like `Uncaught TypeError: window.getSelection(...).focusNode?.closest is not a function`